Resources/doc/reference/configuration/headers.rst

```diff
@@ -332,21 +332,6 @@ then use on the reverse proxy:
 
 This example adds the header ``X-Reverse-Proxy-TTL: 3600`` to your responses.
 Varnish by default knows nothing about this header. To make this solution work,
-you need to extend your varnish ``vcl_fetch`` configuration:
-
-.. code-block:: c
-
-    sub vcl_fetch {
-        if (beresp.http.X-Reverse-Proxy-TTL) {
-            C{
-                char *ttl;
-                ttl = VRT_GetHdr(sp, HDR_BERESP, "\024X-Reverse-Proxy-TTL:");
-                VRT_l_beresp_ttl(sp, atoi(ttl));
-            }C
-            unset beresp.http.X-Reverse-Proxy-TTL;
-        }
-    }
-
-Note that there is a ``beresp.ttl`` field in VCL but unfortunately it can only
-be set to absolute values and not dynamically. Thus we have to revert to a C
-code fragment.
+you need to extend your varnish ``vcl_fetch`` configuration. Please refer to
+the :ref:`FOSHttpCache library’s docs <foshttpcache:varnish_customttl>`
```

Comment:
True, but we decided to not have separate unstable branches. So let’s link both master branches together and have the bundle master’s docs point to the lib’s master ones?

> On 30 Sep 2015, at 09:22, David Buchmann <notifications@github.com> wrote:
> 
> In Resources/doc/reference/configuration/headers.rst <https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/pull/240#discussion_r40764858>:
> 
> > -    sub vcl_fetch {
> > -        if (beresp.http.X-Reverse-Proxy-TTL) {
> > -            C{
> > -                char *ttl;
> > -                ttl = VRT_GetHdr(sp, HDR_BERESP, "\024X-Reverse-Proxy-TTL:");
> > -                VRT_l_beresp_ttl(sp, atoi(ttl));
> > -            }C
> > -            unset beresp.http.X-Reverse-Proxy-TTL;
> > -        }
> > -    }
> > -
> > -Note that there is a ``beresp.ttl`` field in VCL but unfortunately it can only
> > -be set to absolute values and not dynamically. Thus we have to revert to a C
> > -code fragment.
> > +you need to extend your varnish ``vcl_fetch`` configuration. Please refer to
> > +the :ref:`FOSHttpCache library’s docs <foshttpcache:varnish_customttl>`
> the unstable doc of the bundle (aka 2.0) should link with the unstable doc of the library i would say. otherwise as we update, more and more things will get out of sync.
> —
> Reply to this email directly or view it on GitHub <https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/pull/240/files#r40764858>.
>